server/controllers/question.js

```diff
@@ -9,6 +9,7 @@ import { queryPromise } from "../db/database.js";
 // eslint-disable-next-line no-unused-vars
 import { HttpResponseWrapper } from "../global/HttpControllerWrapper.js";
 
+const IMAGES_ROUTE = "files/images";
 /* eslint-disable no-irregular-whitespace */
```

Comment:
I should have added it in my PR, sorry
```suggestion
/* eslint-disable no-irregular-whitespace */ // For non-breaking spaces around French quotes
```